### ISSUE TITLE
Phone number validation

### ISSUE THREAD

**Issue by palaparthi** [twzurkan/team#1]
I think the phone number validation is broken and is not allowing me add any new team members.

<img width="1492" alt="Screenshot 2024-12-20 at 10 05 33 AM" src="https://github.com/user-attachments/assets/060137fe-2e7b-4131-8fc2-9d25480b8cba" />

**Comment by twzurkan** [comment]
It requires a phone number format that includes hyphen and parentheses.

**Comment by palaparthi** [comment]
> It requires a phone number format that includes hyphen and parentheses.

I think a user friendly error message specifying the format would be helpful, is that something you'd be able to add?

**Comment by twzurkan** [comment]
Of course. I will update that today.On Jan 21, 2025, at 11:34 AM, Sree Palaparthi ***@***.***> wrote:﻿

It requires a phone number format that includes hyphen and parentheses.

I think a user friendly error message specifying the format would be helpful, is that something you'd be able to add?

**Comment by twzurkan** [comment]
Hi, I updated the main app and sub module.  It should show better placeholder and error message. Thanks,Tom ZOn Jan 21, 2025, at 12:16 PM, Thomas Zurkan ***@***.***> wrote:﻿Of course. I will update that today.On Jan 21, 2025, at 11:34 AM, Sree Palaparthi ***@***.***> wrote:﻿

It requires a phone number format that includes hyphen and parentheses.

I think a user friendly error message specifying the format would be helpful, is that something you'd be able to add?
